tasks/ohome-group-write.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Set Group Write Permissions on ORACLE_HOME/dbs for all installed ORACLE_HOMEs
+# Executed only if delphix_user is not equal to "oracle"
+
+- name: Find inventory
+  shell: cat /etc/oraInst.loc | grep inventory_loc | awk -F"=" '{print $2}'
+  register: oraInst
+
+- debug:
+    msg: "{{ oraInst.stdout }}"
+    verbosity: 2
+
+- name: Parse inventory
+  shell: cat "{{ oraInst.stdout }}/ContentsXML/inventory.xml" |grep "HOME NAME" | awk '{print $3}' | awk -F"=" '{print $2}'  | tr -d '"'
+  register: existing_dbhome
+
+- debug:
+    msg: "Found ORACLE_HOME - {{ item }}"
+    verbosity: 2
+  with_items: "{{ existing_dbhome.stdout_lines }}"
+
+- name: Setting Group Permission on ORACLE_HOME/dbs
+  file:
+    path: "{{ item }}/dbs"
+    mode: g+w
+  with_items: "{{ existing_dbhome.stdout_lines }}"
```

Comment:
Recommend adding a `when` clause here to skip inventory entries for Oracle Grid homes ("CRS=true") and removed Oracle homes ("REMOVED=anything"). Combining this with the `xml` change above, the code would like something like this:
```
name: Setting Group Permission on ORACLE_HOME/dbs
  file:
    path: "{{ item.HOME.LOC }}/dbs"
    mode: g+w
when: item.HOME.REMOVED is not defined and item.HOME.CRS|default('')|lower != "true"
with_items: "{{ existing_dbhome.matches }}"
```

Reply:
Fixed by improving shell command to 
- name: Parse inventory
  shell: cat "{{ oraInst.stdout }}/ContentsXML/inventory.xml" |grep "HOME NAME" | grep -vi "CRS=" | grep -vi "REMOVED=" | awk '{print $3}' | awk -F"=" '{print $2}'  | tr -d '"'
  register: existing_dbhome

 although not using XML yet, this should workaround the issues with Grid and Removed homes.